vxbot/compose.yml

```
@@ -0,0 +1,9 @@
+services:
+  app:
+    image: ghcr.io/raiga0310/vxbot
+    env_file:
+      - .secret
+    deploy:
+      restart_policy:
+        delay: 5s
+        max_attempts: 5
```

```suggestion
        max_attempts: 5

```

ファイルの最後には必ず改行を加えて欲しい.

---

https://github.com/approvers/containers2/pull/33/commits/96e2503322f50c37a7ebf8d0fbc881729b9dd2ea
で対応